pkgs/shells/bash/5.nix

```diff
@@ -87,8 +91,14 @@ stdenv.mkDerivation rec {
     "bash_cv_termcap_lib=libncurses"
   ] ++ lib.optionals (stdenv.hostPlatform.libc == "musl") [
     "--disable-nls"
+    "--enable-static-link"
```

Comment:
musl != static.  There's dynamically linked musl, and non-musl static builds.  This should check `isStatic`.

Reply:
Thanks! Now checks for `isStatic`!